### DISCUSSION TITLE
Trouble getting FirebaseAuth emulator working on Android device

### DISCUSSION THREAD

**Discussion by ghost** [firebase/flutterfire#5139] [Q&A]
My normal workflow for networking locally is to use chrome on the phone+PC and port forward localhost via chrome://inspect 

This is sufficient for mock APIs so I can do GET/POST/etc.

I have firebase emulator working for EMULATED android devices from AVD. So when I use FirebaseAuth with apps on virtual devices, I know it works.

Where the problems begin is when I try to use FirebaseAuth on a physical device via USB with the method I mentioned in the first few sentences. I have port forwarded localhost:9091 and when I GET that on my device, it returns the Json object with the key “authEmulator”. But I cannot get any FirebaseAuth functions to work.

**Comment by curiosbasant** [comment]
is there any hope, of this issue to work?
